<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceClass</class>
 <widget class="QMainWindow" name="InterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>491</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="admin_tab">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
     <widget class="QLabel" name="add_record_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Add record</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
     <widget class="QLabel" name="location_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="toc_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time of creation:</string>
      </property>
     </widget>
     <widget class="QLabel" name="at_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Access time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="fp_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Footage preview:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="title_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="location_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="toc_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="at_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fp_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_record">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add record</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>151</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="list_label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_record">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_record">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_to_list">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add to list</string>
      </property>
     </widget>
     <widget class="QLabel" name="filter_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter records</string>
      </property>
     </widget>
     <widget class="QLabel" name="filter_location_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="filter_at_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Acces time:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filter_location_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filter_at_edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="filter">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_filepath_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Set user list filepath</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filepath_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="set_filepath">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="view_list">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>View list</string>
      </property>
     </widget>
     <widget class="QPushButton" name="open_list">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>350</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Open list</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>380</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="redo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>380</y>
        <width>93</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Redo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user_tab">
     <attribute name="title">
      <string>User</string>
     </attribute>
     <widget class="QLabel" name="list_label_user">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_to_list_user">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add to list</string>
      </property>
     </widget>
     <widget class="QPushButton" name="open_list_user">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>350</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Open list</string>
      </property>
     </widget>
     <widget class="QListWidget" name="list_user">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>151</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="view_list_user">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>View list</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_filepath_label_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Set user list filepath</string>
      </property>
     </widget>
     <widget class="QLabel" name="filter_location_label_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filepath_edit_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="filter_label_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter records</string>
      </property>
     </widget>
     <widget class="QLabel" name="filter_at_label_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Acces time:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filter_user">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filter_at_edit_user">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="filter_location_edit_user">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="set_filepath_user">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_user">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>380</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="redo_user">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>380</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Redo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="undo_user">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Interface.qrc"/>
 </resources>
 <connections/>
</ui>
